bin/run_tests

```diff
@@ -116,11 +116,15 @@ main() {
                     test_javascript_sdk
                     ;;
                 poet)
+                    test_poet_cli
                     test_poet_common
                     test_poet_core
                     test_poet_families
                     test_poet_simulator
                     ;;
+                poet_cli)
```

Comment:
Do you also want to add poet_cli to list of modules at top of file (between lines 21 and 22)?